pkg/controller/worker/machine_dependencies.go

```diff
@@ -14,12 +14,197 @@
 
 package worker
 
-import "context"
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"sort"
+	"strings"
+
+	"github.com/gardener/gardener-extension-provider-packet/pkg/packet"
+	packetclient "github.com/gardener/gardener-extension-provider-packet/pkg/packet/client"
+	util "github.com/gardener/gardener/extensions/pkg/util"
+	v1beta1constants "github.com/gardener/gardener/pkg/apis/core/v1beta1/constants"
+	"github.com/pkg/errors"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	vpnSeed                               = "vpn-seed"
+	apiServerDeploy                       = v1beta1constants.DeploymentNameKubeAPIServer
+	nodeNetworkEnvVar                     = "NODE_NETWORK"
+	equinixMetalPrivateNetworkAnnotations = "metal.equinix.com/network/4/private"
```

Comment:
How does this work/was this tested? When I try to annotate a `Node` with this key then the API server rejects it:

```
$ k annotate no ip-10-222-27-202.eu-west-1.compute.internal metal.equinix.com/network/4/private=foo
The Node "ip-10-222-27-202.eu-west-1.compute.internal" is invalid: metadata.annotations: Invalid value: "metal.equinix.com/network/4/private": a qualified name must consist of alphanumeric characters, '-', '_' or '.', and must start and end with an alphanumeric character (e.g. 'MyName',  or 'my.name',  or '123-abc', regex used for validation is '([A-Za-z0-9][-A-Za-z0-9_.]*)?[A-Za-z0-9]') with an optional DNS subdomain prefix and '/' (e.g. 'example.com/MyName')
```

Reply:
Fixed. And a separate commit this time. And aligned with what CCM is using.